src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -154,15 +156,15 @@ Configure additional routes in the `routes.extend.include` array. These routes a
 **Default:** `undefined`
 </p>
 
-Configure routes to be excluded in the `routes.extend.include` array. These routes are not automatically deduplicated and will be appended to the existing routes as is.
+Configure routes to be excluded from on-demand rendering in the `routes.extend.include` array. These routes will be prerendered and served statically instead, and will not invoke the SSR function.
```

Comment:
@sarah11918 These doesn't have to be necessarily pages though, it can also be static files, such as images, text files, etc.